scripts/generated/app.ts

```diff
@@ -1502,6 +1512,63 @@ export const ERRORS_STRING_KEYS = {
     WITHDRAW_MUST_SPECIFY_CHAIN: 'ERRORS.WITHDRAW_MODAL.WITHDRAW_MUST_SPECIFY_CHAIN',
 };
 
+export const TOKEN_MIGRATION_STRING_KEYS = {
```

Comment:
`TOKEN_MIGRATION_STRING_KEYS` will need to be exported from `index.ts`

Reply:
done!